applications/StructuralMechanicsApplication/custom_elements/shell_thick_element_3D3N.hpp

```diff
@@ -365,10 +332,10 @@ namespace Kratos
 		void AddBodyForces(CalculationData& data, VectorType& rRightHandSideVector);
 
 		void CalculateAll(MatrixType& rLeftHandSideMatrix,
-			VectorType& rRightHandSideVector,
-			ProcessInfo& rCurrentProcessInfo,
-			const bool LHSrequired,
-			const bool RHSrequired);
+            VectorType& rRightHandSideVector,
+            ProcessInfo& rCurrentProcessInfo,
```

Comment:
btw, make the ProcessInfo const (i think this is "your" function, it is not in the base element class)